### ISSUE TITLE
Installation Stuck - Installing Basset [Bug] 

### ISSUE THREAD

**Issue by georgedobre** [Laravel-Backpack/CRUD#5264]
# Bug report

Installation stuck

### What I did

1. I've created a new cPanel user
2. Upgraded node from v10 to v16
```
[dynwork@hive ~]$ node -v
v16.20.1
```
3. Checked the php version
```
[dynwork@hive ~]$ php -v
PHP 8.1.21 (cli) (built: Jul 26 2023 04:48:43) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.21, Copyright (c) Zend Technologies
```
4. Clean installation of Laravel v10
```
[dynwork@hive ~]$ php artisan about

  Environment ........................................................................................
  Application Name ....................................................................... dynamik.app
  Laravel Version ............................................................................ 10.17.1
  PHP Version ................................................................................. 8.1.21
  Composer Version ............................................................................. 2.5.7
  Environment .................................................................................. local
  Debug Mode ................................................................................. ENABLED
  URL ................................................................................... dynamik.work
  Maintenance Mode ............................................................................... OFF
```
5. Moved Laravel installation to root folder and pointed DocumentRoot from `/public_html` to `/public`
6. Added my credentials to composer
 `composer config http-basic.backpackforlaravel.com xxx.user xxx.token`
7. Required `backpack/crud` through composer
`[dynwork@hive ~]$ composer require backpack/crud`
8. Final step, ran `php artisan backpack:install`
And the output of my console is same for the past hour, stuck at Installing Basset
```
[dynwork@hive ~]$ php artisan backpack:install

   STEP 1  Installing Backpack CRUD.

  Publishing configs, views, routes and assets ................................................... DONE
  Creating users table ........................................................................... DONE
  Creating CheckIfAdmin middleware ............................................................... DONE
  Installing Generators .......................................................................... DONE
  Installing Basset ........................................................................... RUNNING
```

The output for `php artisan backpack:version` is:
```
[dynwork@hive ~]$ php artisan backpack:version
### PHP VERSION:
PHP 8.1.21 (cli) (built: Jul 26 2023 04:48:43) (NTS)
Copyright (c) The PHP Group
Zend Engine v4.1.21, Copyright (c) Zend Technologies

### LARAVEL VERSION:
10.17.1.0

### BACKPACK PACKAGE VERSIONS:
backpack/basset: 1.1.1
backpack/crud: 6.1.4
backpack/generators: v4.0.2
```
But, obviously the installation is not completed.

![Screenshot 2023-08-04 081907](https://github.com/Laravel-Backpack/CRUD/assets/10349279/7f2fc6ca-ebbc-483b-9324-1a365a389e34)

Any pointers?
Thank you

**Comment by pxpm** [comment]
That's very strange. 
We had that issue indeed, but to avoid it getting stuck we moved the check to the end of the installer here: https://github.com/Laravel-Backpack/CRUD/pull/5259

So maybe there is other issue that we haven't catched until now.

You told me you are using cPanel, I suspect I may be some permission issue.

The basset installation should only require the package, and create the symlink.

My guess is that the symlink is failing to create. Can you manually create the symlink with php artisan storage:link and try again?

Sorry for the bad experience. 

Cheers

**Comment by georgedobre** [comment]
@pxpm I highly doubt there is any issues with the cpanel or whm. i have installed it numerous times on the same machine, with the same config and versions.

I'll take a look at the article provided see if it helps. 

Thank you

**Comment by georgedobre** [comment]
I've tried creating the symlink but it was already there
```
[dynwork@hive ~]$ php artisan storage:link
   ERROR  The [public/storage] link already exists.
```

Then i've ran the command again, just in case :D

```
[dynwork@hive ~]$ php artisan backpack:install

   STEP 1  Installing Backpack CRUD.

  Publishing configs, views, routes and assets ........................... DONE
  Creating users table ................................................... DONE
  Creating CheckIfAdmin middleware ....................................... DONE
  Installing Generators .................................................. DONE
  Installing Basset ................................................... RUNNING
```

**Comment by pxpm** [comment]
Hey @georgedobre I've just merged a fix and released a new version 6.1.5. 

Please upgrade your backpack/crud to the latest version and run the installed again. 

We should be good.

We will perform the check at the end of the install command, it may fail, will tell you why, but the installation will be finished by that time. 

Let me know if you are still experiencing issues.

Cheers

**Comment by georgedobre** [comment]
I have updated composer with `composer update` than ran again 
`[dynwork@hive ~]$ php artisan backpack:install`

Everything worked fine. 

Thank you
